bosh/opsfiles/production.yml

```diff
@@ -0,0 +1,75 @@
+- type: replace
+  path: /name?
+  value: shibboleth-production
```

Comment:
Why use an operation for the deployment name? We already have a variable for this.

Reply:
I was just replacing what we had.  I don't know why the original writer did it that way, so I figured that I'd continue to follow their lead.  What variable should we replace it with?  On the face of it, it seems like we ought to be able to just replace it in the manifest with "shibboleth", but maybe there's a reason why it needs to be named staging/prod for some reason?

Reply:
Oh, you are right.  I guess I was just mechanically de-sprucing and didn't notice that.  Thanks!